Source/SwiftLintFramework/Rules/UntypedErrorRule.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  UntypedErrorRule.swift
+//  SwiftLint
+//
+//  Created by Daniel.Metzing on 17/02/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct UntypedErrorRule: ASTRule, OptInRule, ConfigurationProviderRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    private static let regularExpression = regex(
+        "catch" + // The catch keyword
+        "(?:"   + // Start of the first non-capturing group
+        "\\s+"  + // At least one any type of whitespace character
```

Comment:
A whitespace is not required if parentheses are used: `catch(let error) {`. 

Also, the rule currently wouldn't catch when they're used even with spaces.

Reply:
Updated the pattern to capture the errors with parentheses with and without spaces.